File: test-suite-logback/src/test/java/io/micronaut/logback/Application.java
package io.micronaut.logback;

import io.micronaut.runtime.Micronaut;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class Application {

    private static final Logger LOG = LoggerFactory.getLogger(Application.class);

    public static void main(String[] args) {
        LOG.trace("starting the app");
        Micronaut.run(Application.class, args);
    }
}

File: examples/hello-world-kotlin/src/main/kotlin/example/Application.kt
package example

import io.micronaut.runtime.Micronaut

object Application {

    @JvmStatic
    fun main(args: Array<String>) {
        System.setProperty("org.slf4j.simpleLogger", "TRACE")
        Micronaut.build().packages(Package.getPackage("example")).start(javaClass)
    }
}


